DuckDuckGoTests/OmniBarStateTests.swift

```diff
@@ -83,9 +83,9 @@ class OmniBarStateTests: XCTestCase {
         XCTAssertTrue(testee.showSearchLoupe)
     }
 
-    func testWhenEnteringHomeTextEditingStateThenTextIsCleared() {
+    func testWhenEnteringHomeTextEditingStateThenTextIsNotCleared() {
         let testee = HomeTextEditingState()
-        XCTAssertTrue(testee.clearTextOnStart)
+        XCTAssertFalse(testee.clearTextOnStart)
```

Comment:
@brindy Was there any particular reason why this was made this way in the first place? I tried to break this after the change and it seems to work - but I'm afraid I've missed something.